Backpack/src/main/java/net/skyscanner/backpack/calendar/view/WeekdayHeaderView.kt

```diff
@@ -30,25 +31,21 @@ internal class WeekdayHeaderView @JvmOverloads constructor(
   }
 
   internal fun initializeWithLocale(locale: Locale) {
-    val simpleDateFormat = SimpleDateFormat("ccc", locale)
-    val calendar = Calendar.getInstance(locale)
-    val weekStart = calendar.firstDayOfWeek
-    val numberOfDaysInWeek = 7
-
-    for (i in 0 until numberOfDaysInWeek) {
-      val calendarDay = (i + weekStart) % numberOfDaysInWeek
-      calendar.set(Calendar.DAY_OF_WEEK, calendarDay)
-      val calendarDayName = simpleDateFormat.format(calendar.time)
-
-      when (i) {
-        0 -> firstWeekdayView.text = calendarDayName
-        1 -> secondWeekdayView.text = calendarDayName
-        2 -> thirdWeekdayView.text = calendarDayName
-        3 -> fourthWeekdayView.text = calendarDayName
-        4 -> fifthWeekdayView.text = calendarDayName
-        5 -> sixthWeekdayView.text = calendarDayName
-        6 -> seventhWeekdayView.text = calendarDayName
-      }
-    }
+    val formatter = DateTimeFormatter.ofPattern("ccc", locale)
+    dayOfWeek(locale, formatter, 1)
```

Comment:
What is this line for?

Reply:
Should have been removed, just a leftover. Cleaning it up now.